MicrosoftFluentUI.podspec

```diff
@@ -287,6 +293,7 @@ fi', :execution_position => :before_compile }
 
   s.subspec 'TableView_ios' do |tableview_ios|
     tableview_ios.platform = :ios
+    tableview_ios.dependency 'MicrosoftFluentUI/Divider_ios'
     tableview_ios.dependency 'MicrosoftFluentUI/Label_ios'
     tableview_ios.dependency 'MicrosoftFluentUI/Separator_ios'
```

Comment:
Would it be possible to clean up the Separator at the same time, or should that happen in another PR?

Reply:
I think we can tackle it in a separate work item. This one is already too big/complex.

Reply:
Resolved offline; we're removing divider in this PR and leaving separator in for the time being